students/DrewSmith/lessons/lesson03/assignment/basic_operations.py

```diff
@@ -0,0 +1,86 @@
+'''
+Basic operations for interacting with customers database
+'''
+
+
+import logging
+from peewee import DoesNotExist
+from assignment.customers_model import Customer
+
+def add_customer(customer_id, name, lastname, home_address,
+                 phone_number, email_address, status, credit_limit):
+    ''' Add a customer '''
+    try:
+        credit_limit = float(credit_limit)
+    except ValueError as err:
+        logging.error(err)
+        raise
+
+    if not isinstance(status, bool):
+        logging.error(f"Invalid value: status is not a bool: {status}")
+        raise ValueError(f"Invalid value: status is not a bool: {status}")
```

Comment:
Nice. I like f strings. Note you are writing the same thing twice. You could set it to a variable instead. This is the sort of thing we keep in a config file, that way you change it one place and everyone sees it, including tests.